dnet/dhcp/dhcp.go

```diff
@@ -41,8 +41,11 @@ type LanSpec struct {
 }
 
 func createDHCPFile(nets Networks) string {
+
+
 	var tpl bytes.Buffer
-	tmpl := template.Must(template.ParseFiles("/home/ubuntu/defat/dnet/dhcp/dhcpd.conf.tmpl"))
+	//TODO: Get path from location of running
+	tmpl := template.Must(template.ParseFiles("/home/ubuntu/vlad/sec03/defatt/dnet/dhcp/dhcpd.conf.tmpl"))
```

Comment:
Could be done in better way like getting the path directly instead of providing static path. Here is the code snippet that you may use: 
```go

dir, err := os.Getwd()   // get working directory 
if err !=nil {
    // here log or return the error
}
 fmt.Printf("Current directory is  [ %s ]\n ",dir)

```
`dir` variable most probably will be `/home/ubuntu/vlad/sec03/defatt` in your case, since you will execute the program from `/home/ubuntu/vlad/sec03/defatt`  dir.  Afterwards you can concatenate rest which is `/dnet/dhcp/dhcpd.conf.tmpl`

At the end, it should like this, 

```go
dir, err := os.Getwd()   // get working directory 
if err !=nil {
    // here log or return the error
}
fullPathToTemplate := fmt.Sprintf("%s%s", dir, '/dnet/dhcp/dhcpd.conf.tmpl')

tmpl := template.Must(template.ParseFiles(fullPathToTemplate))

```

In this case, you do not need to change the path each time that you changed your working directory.

Reply:
It will be implemented in the next commit. 

Useful snippet  💪🏼